packages/cli/src/commands/new/project.ts

```diff
@@ -102,7 +102,7 @@ const getProviderPackageName = async (prompter: Prompter, providerPackageName?:
   }
 }
 
-const parseConfig = async (
+export const parseConfig = async (
```

Comment:
until we have a `--default` or something like that, I had to stub the returned value from here